code/modules/mob/living/carbon/human/life.dm

```diff
@@ -24,9 +33,6 @@
 	if(stat == DEAD)
 		handle_decay()
 
-	if(life_tick > 5 && timeofdeath && (timeofdeath < 5 || world.time - timeofdeath > 6000))	//We are long dead, or we're junk mobs spawned like the clowns on the clown shuttle
```

Comment:
Pretty sure @Necaladun will be the only one familiar with this bug; there was this nonsense where some pre-spawned dead clowns on the crashed shuttle in space would eventually cause massive amounts of lag due to constnatly taking brute+burn damage.

The source of the problem was a combination of old style medical huds and Bay's awful wound datums. Both of these are purged---so I"m not worried about this anymore.